server/src/test/java/org/elasticsearch/index/mapper/GenericStoreDynamicTemplateTests.java

```diff
@@ -51,16 +51,13 @@ public void testSimple() throws Exception {
         assertThat(f.stringValue(), equalTo("some name"));
         assertThat(f.fieldType().stored(), equalTo(true));
 
-        FieldType fieldType = mapperService.getLuceneFieldType("name");
+        FieldType fieldType = mapperService.fieldType("name").getTextSearchInfo().getLuceneFieldType();
         assertThat(fieldType.stored(), equalTo(true));
 
         boolean stored = false;
         for (IndexableField field : doc.getFields("age")) {
             stored |=  field.fieldType().stored();
         }
         assertTrue(stored);
-
```

Comment:
`TextSearchInfo.NUMERIC` always returns `stored` as `false`.  I thought about making this configurable, but the only place in the code that uses this is in highlighting, and numeric fields can't be highlighted, so I think it's OK to be slightly incorrect here.

Reply:
does it make sense to add this as a comment somewhere?

Reply:
I added a comment to `TextSearchInfo.NUMERIC` javadocs